### PR TITLE
[zh-cn]: Update translation of Function.arguments()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#15403

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md [hunk 1]
```diff
@@ -1,57 +1,90 @@
...
 ## 描述
 
-`function.arguments` 已经被废弃了，现在推荐的做法是使用函数内部可用的 {{jsxref("Functions/arguments", "arguments")}} 对象来访问函数的实参。
+`arguments` 的值是一个类数组对象，对应于传递入函数的参数。
+
+在递归的情况下，即如果函数 `f` 在调用栈上出现多次，`f.arguments` 的值表示最近一次调用该函数时的参数。
+
+如果函数没有正在进行的未完成的调用（即函数已被调用但尚未返回），`arguments` 属性的值通常为 {{jsxref("Operators/null", "null")}}。
+
+请注意，ECMAScript 规范只规定了 `Function.prototype` 具有一个初始的 `arguments` 访问器，对于任何 `get` 或 `set` 请求都无条件地抛出 {{jsxref("TypeError")}}（称为“毒丸访问器”），而且实现不允许改变此语义，除非是非严格的普通函数。`arguments` 属性的实际行为如果不是抛出错误，就是由实现定义的。例如，Chrome 将其定义为自有的数据属性，而 Firefox 和 Safari 将初始的毒丸 `Function.prototype.arguments` 访问器扩展为特殊处理非严格函数的 `this` 值。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md:20]:
```suggestion
请注意，ECMAScript 规范只规定了 `Function.prototype` 具有一个初始的 `arguments` 访问器，对于任何 `get` 或 `set` 请求都无条件地抛出 {{jsxref("TypeError")}}（称为“毒丸访问器”），而且实现不允许改变此语义，除非是非严格的普通函数。`arguments` 属性的实际行为如果不是抛出错误，就是由实现定义的。例如，Chrome 将其定义为自有的数据属性，而 Firefox 和 Safari 将初始的毒丸访问器 `Function.prototype.arguments` 扩展为特殊处理非严格函数的 `this` 值。
```

File: files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md [hunk 2]
```diff
@@ -1,57 +1,90 @@
...
+```js
+(function f() {
+  if (Object.hasOwn(f, "arguments")) {
+    console.log(
+      "arguments 是带有描述符的自有属性",
+      Object.getOwnPropertyDescriptor(f, "arguments")
+    );
+  } else {
+    console.log(
+      "f 没有自有的名 为arguments 的属性。尝试获取 f.[[Prototype]].arguments"
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md:31]:
```suggestion
      "f 没有自有的名为 arguments 的属性。尝试获取 f.[[Prototype]].arguments"
```

File: files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md [hunk 3]
```diff
@@ -1,57 +1,90 @@
...
+  }
+})();
 
-在函数递归调用的时候（在某一刻同一个函数运行了多次，也就是有多套实参），那么 `arguments` 属性的值是最近一次该函数调用时传入的实参，下面的示例有演示。
+// 在 Chrome 中：
+// arguments 是带有描述符的自有属性 {value: Arguments(0), writable: false, enumerable: false, configurable: false}
 
-如果函数不在执行期间，那么该函数的 `arguments` 属性的值是 `null`。
+// 在 Firefox 中：
+// f 没有自有的名 为arguments 的属性。尝试获取 f.[[Prototype]].arguments
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md:46]:
```suggestion
// f 没有自有的名为 arguments 的属性。尝试获取 f.[[Prototype]].arguments
```

**Comment by yin1999** [comment]
Please run `yarn install` to update the toolchain

**Comment by JasonLamv-t** [comment]
> Please run `yarn install` to update the toolchain

Got it.

File: files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md [hunk 4]
```diff
@@ -1,57 +1,90 @@
...
-title: Function.arguments
+title: Function.prototype.arguments
 slug: Web/JavaScript/Reference/Global_Objects/Function/arguments
 ---
 
-{{JSRef}} {{deprecated_header}}
+{{JSRef}}{{Deprecated_Header}}{{Non-standard_Header}}
 
-`function.arguments` 属性代表传入函数的实参，它是一个类数组对象。
+**备注：** {{jsxref("Function")}} 对象的 `arguments` 属性已被弃用。访问 `arguments` 对象的推荐方式是在函数内部引用可用的变量 {{jsxref("Functions/arguments", "arguments")}}。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md:8]:
```suggestion
> **备注：** {{jsxref("Function")}} 对象的 `arguments` 属性已被弃用。访问 `arguments` 对象的推荐方式是在函数内部引用可用的 {{jsxref("Functions/arguments", "arguments")}} 变量。
```

File: files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md [hunk 5]
```diff
@@ -1,57 +1,90 @@
...
 ## 描述
 
-`function.arguments` 已经被废弃了，现在推荐的做法是使用函数内部可用的 {{jsxref("Functions/arguments", "arguments")}} 对象来访问函数的实参。
+`arguments` 的值是一个类数组对象，对应于传递入函数的参数。
+
+在递归的情况下，即如果函数 `f` 在调用栈上出现多次，`f.arguments` 的值表示最近一次调用该函数时的参数。
+
+如果函数没有正在进行的未完成的调用（即函数已被调用但尚未返回），`arguments` 属性的值通常为 {{jsxref("Operators/null", "null")}}。
+
+请注意，ECMAScript 规范只规定了 `Function.prototype` 具有一个初始的 `arguments` 访问器，对于任何 `get` 或 `set` 请求都无条件地抛出 {{jsxref("TypeError")}}（称为“毒丸访问器”），而且实现不允许改变此语义，除非是非严格的普通函数。`arguments` 属性的实际行为如果不是抛出错误，就是由实现定义的。例如，Chrome 将其定义为自有的数据属性，而 Firefox 和 Safari 将初始的毒丸访问器 `Function.prototype.arguments` 扩展为特殊处理非严格函数的 `this` 值。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md:20]: I'm confused about the translation of these two paragraphs. Maybe we can improve them.

Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md:20]:
```suggestion
`arguments` 的值是一个类数组对象，对应于传入函数的参数。

在递归的情况下，即如果函数 `f` 在调用栈上出现多次，`f.arguments` 的值表示最近一次调用该函数时的参数。

如果函数没有正在进行的、未完成的调用（即函数已被调用但尚未返回），`arguments` 属性的值通常为 {{jsxref("Operators/null", "null")}}。

请注意，ECMAScript 规范只规定了 `Function.prototype` 具有一个初始的 `arguments` 访问器，对于任何 `get` 或 `set` 请求都无条件地抛出 {{jsxref("TypeError")}}（称为“毒丸访问器”），而且引擎实现不允许改变此语义，除非是非严格的普通函数。（对于非严格的普通函数）`arguments` 属性的实际行为如果不是抛出错误，这该行为由实现定义。例如，Chrome 将其定义为自有的数据属性，而 Firefox 和 Safari 扩展了初始的毒丸访问器 `Function.prototype.arguments` ，以特殊处理非严格函数的 `this` 值。
```

File: files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md [hunk 6]
```diff
@@ -1,57 +1,90 @@
...
 ## 描述
 
-`function.arguments` 已经被废弃了，现在推荐的做法是使用函数内部可用的 {{jsxref("Functions/arguments", "arguments")}} 对象来访问函数的实参。
+`arguments` 的值是一个类数组对象，对应于传入函数的参数。
+
+在递归的情况下，即如果函数 `f` 在调用栈上出现多次，`f.arguments` 的值表示最近一次调用该函数时的参数。
+
+如果函数没有正在进行的、未完成的调用（即函数已被调用但尚未返回），`arguments` 属性的值通常为 {{jsxref("Operators/null", "null")}}。
+
+请注意，ECMAScript 规范只规定了 `Function.prototype` 具有一个初始的 `arguments` 访问器，对于任何 `get` 或 `set` 请求都无条件地抛出 {{jsxref("TypeError")}}（称为“毒丸访问器”），而且引擎实现不允许改变此语义，除非是非严格的普通函数。（对于非严格的普通函数）`arguments` 属性的实际行为如果不是抛出错误，这该行为由实现定义。例如，Chrome 将其定义为自有的数据属性，而 Firefox 和 Safari 扩展了初始的毒丸访问器 `Function.prototype.arguments` ，以特殊处理非严格函数的 `this` 值。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md:20]:
```suggestion
请注意，ECMAScript 规范只规定了 `Function.prototype` 具有一个初始的 `arguments` 访问器，对于任何 `get` 或 `set` 请求都无条件地抛出 {{jsxref("TypeError")}}（称为“毒丸访问器”），而且引擎实现不允许改变此语义，除非是非严格的普通函数。（对于非严格的普通函数）`arguments` 属性的实际行为如果不是抛出错误，则该行为由实现定义。例如，Chrome 将其定义为自有的数据属性，而 Firefox 和 Safari 扩展了初始的毒丸访问器 `Function.prototype.arguments` ，以特殊处理非严格函数的 `this` 值。
```

File: files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md [hunk 7]
```diff
@@ -1,57 +1,90 @@
...
 ## 描述
 
-`function.arguments` 已经被废弃了，现在推荐的做法是使用函数内部可用的 {{jsxref("Functions/arguments", "arguments")}} 对象来访问函数的实参。
+`arguments` 的值是一个类数组对象，对应于传入函数的参数。
+
+在递归的情况下，即如果函数 `f` 在调用栈上出现多次，`f.arguments` 的值表示最近一次调用该函数时的参数。
+
+如果函数没有正在进行的、未完成的调用（即函数已被调用但尚未返回），`arguments` 属性的值通常为 {{jsxref("Operators/null", "null")}}。
+
+请注意，ECMAScript 规范只规定了 `Function.prototype` 具有一个初始的 `arguments` 访问器，对于任何 `get` 或 `set` 请求都无条件地抛出 {{jsxref("TypeError")}}（称为“毒丸访问器”），而且引擎实现不允许改变此语义，除非是非严格的普通函数。（对于非严格的普通函数）`arguments` 属性的实际行为如果不是抛出错误，则该行为由实现定义。例如，Chrome 将其定义为自有的数据属性，而 Firefox 和 Safari 扩展了初始的毒丸访问器 `Function.prototype.arguments` ，以特殊处理非严格函数的 `this` 值。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/function/arguments/index.md:20]:
```suggestion
请注意，ECMAScript 规范只规定了 `Function.prototype` 具有一个初始的 `arguments` 访问器，对于任何 `get` 或 `set` 请求都无条件地抛出 {{jsxref("TypeError")}}（称为“毒丸访问器”），而且引擎实现不允许改变此语义，除非是非严格的普通函数。（对于非严格的普通函数）`arguments` 属性的实际行为如果不是抛出错误，则该行为由实现定义。例如，Chrome 将其定义为自有的数据属性，而 Firefox 和 Safari 扩展了初始的毒丸访问器 `Function.prototype.arguments`，以特殊处理非严格函数的 `this` 值。
```

**Review by yin1999 — APPROVED** [review]
LGTM. Thank you @JasonLamv-t

**Pull request merged**